### PR TITLE
Settings UI

### PR DESCRIPTION
**Pull request by SvetaMemphis**

### CODE REVIEW

File: ui_src/src/components/radioButton/index.js [hunk 1]
```diff
@@ -46,9 +46,33 @@ const RadioButton = (props) => {
                 defaultValue={radioValue || options[0]?.value}
             >
                 {options.map((option) => (
-                    <Radio key={option.id} value={option.value}>
-                        <span style={{ fontFamily: fontFamily }}>{option.label}</span>
-                    </Radio>
+                    <div
+                        style={{
+                            background: props.labelType && radioValue === option.value ? 'rgba(101, 87, 255, 0.1)' : '',
```
Comment (avrhamNeeman) [inline ui_src/src/components/radioButton/index.js:51]: Please try to extract the style to variable

File: ui_src/src/domain/settings/profile/imgLoader/index.js [hunk 2]
```diff
@@ -1,149 +0,0 @@
-// Copyright 2021-2022 The Memphis Authors
```
Comment (avrhamNeeman) [inline ui_src/src/domain/settings/profile/imgLoader/index.js:1]: Why you delete this file and did not use it?

File: ui_src/src/domain/settings/profile/index.js [hunk 3]
```diff
@@ -63,31 +151,139 @@ function Profile() {
...
+                    fontSize="14px"
+                    fontWeight="600"
+                    aria-haspopup="true"
+                    disabled={!checkboxdeleteAccount || userName === 'root'}
+                    onClick={() => modalFlip(true)}
+                />
+            </div>
             <Modal
-                header="Remove user"
+                header="Remove accont"
```
Comment (avrhamNeeman) [inline ui_src/src/domain/settings/profile/index.js:253]: account

File: ui_src/src/domain/settings/profile/imgLoader/index.js (hunk 2 above)
Comment (SvetaMemphis) [reply ui_src/src/domain/settings/profile/imgLoader/index.js:1]: Done

File: ui_src/src/components/radioButton/index.js (hunk 1 above)
Comment (SvetaMemphis) [reply ui_src/src/components/radioButton/index.js:51]: done

**Pull request merged**